frontend/src/components/Util/AppInit/LoginBarrier.tsx

```diff
@@ -60,8 +61,9 @@ export default function LoginBarrier({ appRenderer }: Props): ReactElement {
           setLoaded(false);
           return;
         }
-        const { email } = currentUserFromFirebase;
+        const { email, displayName } = currentUserFromFirebase;
         if (email.endsWith('@cornell.edu')) {
+          addUserInfo(email, displayName);
```

Comment:
while you are at it, could you make it store the profile pic as well, since the design requires it.

Reply:
Done. will update the PR desc accordingly